AmazonChimeSDK/AmazonChimeSDK/internal/audio/DefaultAudioClientController.swift

```diff
@@ -161,4 +161,8 @@ extension DefaultAudioClientController: AudioClientController {
         }
```

Comment:
why not add `NotificationCenter.default` here?

Reply:
Resolved.